cmd/soroban-cli/src/commands/contract/info.rs

```diff
@@ -0,0 +1,72 @@
+use std::fmt::Debug;
+
+pub mod env_meta;
+pub mod interface;
+pub mod meta;
+mod shared;
+
+#[derive(Debug, clap::Subcommand)]
+pub enum Cmd {
+    /// Output the env required metadata stored in a contract.
+    ///
+    /// Env-meta is information stored in all contracts, in the
+    /// `contractenvmetav0` WASM custom section, about the environment
+    /// that the contract was built for. Env-meta allows the Soroban Env
+    /// to know whether the contract is compatibility with the network in
+    /// its current configuration.
+    ///
+    /// The data outputted by this command is a stream of `SCEnvMetaEntry` XDR values.
+    /// See the type definitions in [stellar-xdr](https://github.com/stellar/stellar-xdr).
+    /// [See also XDR data format](https://developers.stellar.org/docs/learn/encyclopedia/data-format/xdr).
+    ///
+    /// Outputs no data when no data is present in the contract.
+    EnvMeta(env_meta::Cmd),
```

Comment:
So that these commands are ordered by importantance to the average dev, I'd put Interface first, then Meta, then EnvMeta.